src/components/chatbot.tsx

```diff
@@ -40,10 +51,10 @@ const StyledChatbot = styled(ReactSimpleChatbot)`
 
   .rsc-content {
     padding: 15px;
-    height: calc(100% - 30px);
+    height: calc(100% - ${props => (props.showInput ? 81 : 30)}px);
```

Comment:
could we have a const for the input height? 
```
const INPUT_HEIGHT = 51
height: calc(100% - ${props => (props.showInput && INPUT_HEIGHT) + 30} px);
```

Reply:
yes we can, almost did it initially but didn't know if overkill :p

Reply:
done